packages/jellyfish-testing/src/testing.ts

```diff
@@ -6,24 +6,32 @@ import { TestingRawTx } from './rawtx'
 import { TestingICX } from './icxorderbook'
 import { TestingMisc } from './misc'
 import { TestingGroupAnchor } from './anchor'
-import { ContainerGroup, MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontainers'
+import { ContainerGroup, DeFiDContainer, MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontainers'
 import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
 import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
 
 export class Testing {
+  public readonly container: MasterNodeRegTestContainer
   public readonly fixture = new TestingFixture(this)
-  public readonly token = new TestingToken(this.container, this.rpc)
-  public readonly poolpair = new TestingPoolPair(this.container, this.rpc)
-  public readonly rawtx = new TestingRawTx(this.container, this.rpc)
+  public readonly token!: TestingToken
+  public readonly poolpair!: TestingPoolPair
+  public readonly rawtx!: TestingRawTx
   public readonly icxorderbook = new TestingICX(this)
```

Comment:
let's move `icxorderbook` and `fixture` also inside the `MasterNodeRegTestContainer` check.
even though it does not take the container as input param, it's already there in `Testing`.
so for non children of `MasterNodeRegTestContainer` will not have these defined. 
But if it wants to call any of the functionality, it has to call it via `testing.rpc.*` or `testing.container.call()`. responsibility of getting successful or not is up to the caller.